src/main/java/com/cse/helium/app/controllers/Controller.java

```diff
@@ -39,26 +38,25 @@ public Object getAll(
 
     Map<String,Object> queryParams = new HashMap<>();
     HttpHeaders headers = new HttpHeaders();
-    headers.setContentType(MediaType.TEXT_PLAIN);
+    headers.setContentType(MediaType.APPLICATION_JSON);
 
     if (query.isPresent()) {
       if (Boolean.TRUE.equals(validator.isValidSearchQuery(query.get()))) {
         queryParams.put("q", query.get().trim().toLowerCase().replace("'", "''"));
       } else {
-        logger.error(Constants.INVALID_Q_PARAM_MSG);
+        logger.error("INVALID_QUERY_MESSAGE");
```

Comment:
I think this can still be used as a constant, rather than inlining the string as such. Same goes for the `pagenum_message` and the other messages.

Reply:
the value of the constant is very large so not sure if it should be logged each time. would you rather i make a new set of constants for logging purposes?

Reply:
I think you can modify the `*PARAM_MSG` constants that were used before.

Reply:
redid those error logs to line up with 404 logs. more detail as well